src/org/labkey/test/components/react/Pager.java

```diff
@@ -63,8 +63,11 @@ public int getPageSize()                // only works on GridPanel
     public Pager clickPrevious()
     {
         if (elementCache().prevButton().isPresent())
-            _pagedComponent.doAndWaitForUpdate(()->
+        {
+            getWrapper().scrollIntoView(elementCache().prevButton().get());
+            _pagedComponent.doAndWaitForUpdate(() ->
                     elementCache().prevButton().get().click());
+        }
```

Comment:
We're finding the button three times here. We should avoid that.
```suggestion
        elementCache().prevButton().ifPresent(button -> {
            _pagedComponent.doAndWaitForUpdate(()-> {
                getWrapper().scrollIntoView(button);
                button.click();
            });
        });
```

Reply:
I was unaware of the ifPresent option.
I have made the update, but it does touch on an issue we discussed in our coding standards meeting. If the previous, or next button, is not there this code will do nothing. Does that run counter to what we had talked about?